src/scoping-shim.js

```diff
@@ -512,7 +512,14 @@ export default class ScopingShim {
   // any necessary ShadyCSS static and property based scoping selectors
   setElementClass(element, classString) {
     let root = StyleUtil.wrap(element).getRootNode();
-    let classes = classString ? classString.split(/\s/) : [];
+    let classes = [];
+    if( classString) {
+      if (Array.isArray(classString) {
+        classes = classString;
```

Comment:
This doesn't get us native behavior:

```js
div.className = ['a', 'b'];
div.className; // "a,b"
```

The correct fix here is to just always use `String(classString)`. This also gets us the correct behavior for these other cases that the current code fails or gives incorrect results on:

```js
div.className = false; // "false"
div.className = true; // "true"
div.className = undefined; // "undefined"
```